packages/gatsby-remark-images/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -288,6 +288,46 @@ exports[`it transforms images in markdown with query strings 1`] = `
     </span>"
 `;
 
+exports[`it transforms images in markdown with the "withAvif" option 1`] = `
+"<span
+      class=\\"gatsby-resp-image-wrapper\\"
+      style=\\"position: relative; display: block; margin-left: auto; margin-right: auto; max-width: 300px; \\"
+    >
+      <a
+    class=\\"gatsby-resp-image-link\\"
+    href=\\"not-a-real-dir/images/my-image.jpeg\\"
+    style=\\"display: block\\"
+    target=\\"_blank\\"
+    rel=\\"noopener\\"
+  >
+    <span
+    class=\\"gatsby-resp-image-background-image\\"
+    style=\\"padding-bottom: 133.33333333333331%; position: relative; bottom: 0; left: 0; background-image: url('data:image/png;base64,iVBORw'); background-size: cover; display: block;\\"
+  ></span>
+  <picture>
+          <source
+              srcset=\\"not-a-real-dir/images/my-image.jpeg, not-a-real-dir/images/my-image.jpeg\\"
```

Comment:
shouldn't there be an avif image here?

Reply:
No, it's okay, because jest mocked fluid function here.
https://github.com/gatsbyjs/gatsby/blob/46606d3dd7b10ab87638c599dd021561b48ddaef/packages/gatsby-remark-images/src/__tests__/index.js#L7

This function just returns the string with absolute paths in it, without format transform.